boot/src/main/java/module-info.java

```diff
@@ -1,4 +1,4 @@
-module edpn.boot {
+module io.edpn.backend.boot {
```

Comment:
module name != package name. It can cause confusion if these are named exactly the same.